object.js

```diff
@@ -0,0 +1,48 @@
+//point2
+let favoriteMovie= {
+title : "The Lord Of The Rings",
+year: 2003,
+director: "Peter Jackson",
+actors: ["Vigoo Mortensen","Sean Astin"],
+rating: 8.9,
+timewatched: 20,
+//point3
+recommend(){
+    console.log("this movie is the best movie ever because it can take you to another world full with suspense and action and some emotions ")
+},
+watch(){
+    this.timewatched +=1
+},
+setRating(newRating){
+    this.rating=newRating
+},
+addActor(actorName){
+    // this.actors.push(actorName)
+    this.actors=[...this.actors, actorName]
+},
+isPopular(){
+return this.rating>=8
+},
+summary(){
+    console.log(this)
```

Comment:
🤨 Really? this is gonna return the whole object to me, I want just a brief description of this movie